jdbcSource/src/main/java/io/vantiq/extsrc/jdbcSource/JDBCCore.java

```diff
@@ -253,6 +262,8 @@ public synchronized void executePolling(String pollQuery) {
             }
         } catch (VantiqSQLException e) {
             log.error("Could not execute polling query.", e);
+        } catch (Exception e) {
+            log.error("An unexpected error occured when executing the polling query.");
```

Comment:
For all of these, we need to include whatever data we need to debug.  So, we should log the error we got (not just "oops, something happened."  Where we can, the error should include the request that precipitated it and the response we got that we failed to deal with.